src/content/manifest/environment-variables.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: Environment variables
+description: Use Okteto's built-in environment variables to streamline your Okteto manifest.
+sidebar_label: Environment variables
+id: environment-variables
+---
+
+The Okteto manifest utilizes environment variables with each section supporting specific ENV VARs. Additional ENV VARs are added regularly and it's important to understand the existing ones and their usage. Currently, there are three types of ENV VARs to be aware of:
+
+## Using environment variables
+
+In order to use environment variables within your Okteto manifest you must add them to `$OKTETO_ENV` within your `deploy` section. This will make the environment variables available in all following manifest sections (e.g. `destroy`, `dev`) as well as in the Okteto UI. To do this you’ll add something similar to this in your `deploy` section (using your actual variable names):
+
+```yaml
+{
+  echo "OKTETO_EXTERNAL_S3_ENDPOINTS_BUCKET_URL=$s3Dashboard"
+  echo "S3_BUCKET_NAME=$resourceName"
+  echo "OKTETO_EXTERNAL_SQS_ENDPOINTS_QUEUE_URL=$queueDashboard"
+  echo "SQS_QUEUE_URL=$queueUrl"
+  echo "SQS_QUEUE_NAME=$resourceName"
+} >> "$OKTETO_ENV"
+```
+
+### **Default Environment Variables**
+
+These are automatically populated and can be referenced in your Okteto manifest’s `deploy`/`destroy` commands:
```

Comment:
You can also use it in the YAML of the okteto manifest

Reply:
@pchico83 If we remove the word "commands", would that resolve this? I interpret your comment to mean that   these vars can be used within the actual `commands:` but also generally anywhere within the `deploy`/`destroy` sections.

This was copy/pasted from the existing content, but I'm thinking simply removing "commands" would suffice here. WDYT?